frontend/locales/fr.ts

```diff
@@ -1,7 +1,864 @@
+                unstable: 'Instable',
+                recommended: 'Recommandé',
+                forumPost: 'Poste Forum',
+                release: {
+                    bulletin: 'Bulletin de la version',
+                    desc: "Quoi de neuf dans cette version ?",
+                },
+                platforms: 'Plateformes',
+                dependencies: 'Dépendances des plugins',
+            },
+            error: {
+                metaNotFound: 'Impossible de charger les métadonnées à partir du fichier téléversé',
+                jarNotFound: "Impossible d'ouvrir le fichier jar",
+                fileExtension: 'Extension du fichier incorrecte',
+                unexpected: 'Une erreur inattendue est survenue',
+                invalidVersionString: 'Chaîne de version invalide trouvée',
+                duplicateNameAndPlatform: 'Une version avec ce nom compatible avec cette plateforme existe déjà',
+                invalidNumOfPlatforms: 'Nombre de plateforme invalide',
+                duplicate: 'Une version avec ce fichier existe déjà',
+                noFile: 'Impossible de trouver le fichier téléversé',
+                mismatchedFileSize: 'Les tailles du fichier ne correspondent pas',
+                hashMismatch: 'Les hachages du fichier ne correspondent pas',
+                invalidPlatformVersion: 'Version de Minecraft invalide pour la plateforme spécifiée',
+                fileIOError: "Erreur d'E/S du fichier",
+                unknown: 'Une erreur inconnue est survenue',
+                incomplete: 'Fichier {0} du plugin manquant',
+                noDescription: 'Doit contenir une description',
+                invalidPluginDependencyNamespace: 'La dépendance du plugin déclarée a un espace de nom invalide',
+                channel: {
+                    noName: 'Doit contenir un nom de canal spécifié',
+                    noColor: 'Doit contenir une couleur spécifiée pour le canal',
+                },
+            },
+        },
+        edit: {
+            platformVersions: 'Éditer les versions de la plateforme : {0}',
+            pluginDeps: 'Éditer les dépendances du plugin : {0}',
+            error: {
+                noPlatformVersions: 'Doit fournir au moins une version de plateforme valide',
+                invalidVersionForPlatform: '{0} est une version invalide pour {1}',
+                invalidProjectNamespace: "{0} n'est pas un espace de noms de projet valide",
+            },
+        },
+        page: {
+            subheader: '{0} diffusa cette version le {1}',
+            dependencies: 'Dépendances',
+            platform: 'Plateforme',
+            required: '(requis)',
+            adminMsg: '{0} approuva cette version le {1}',
+            reviewLogs: "Journaux d'examen",
+            reviewStart: "Commencer l'examen",
+            setRecommended: 'Définir comme recommandée',
+            setRecommendedTooltip: 'Définir cette version comme étant recommandée pour {0} plateforme',
```

Comment:
```suggestion
            setRecommendedTooltip: 'Définir cette version comme étant recommandée pour la plateforme {0}',
```
The name of the platform should be after.